util/perl/OpenSSL/config.pm

```diff
@@ -872,8 +872,11 @@ EOF
       # model (PUT, SPT or nothing), target execution environment (OSS or
       # GUARDIAN).  And still, there must be some kind of default when
       # nothing else is said.
+      #
+      # nsv is a virtual x86 environment, equivalent to nsx, so we enforce
+      # the latter.
       [ 'nse-tandem-nsk.*',       { target => 'nonstop-nse' } ],
```

Comment:
Should be
      [ 'NSE-L-.*-NONSTOP_KERNEL.*', { target => 'nonstop-nse' } ],

Reply:
The error was actually higher up...  I had switched around the interpretation of uname values.  Just pushed a fixup